index.d.ts

```diff
@@ -0,0 +1,49 @@
+declare type Acl = 'private' | 'bucketOwnerRead' | 'bucketOwnerFullControl' | 'projectPrivate' | 'authenticatedRead' | 'publicRead' | 'publicReadWrite';
+
+declare interface MulterOptions {
```

Comment:
some properties are multiple types. I think you use union type. for example: [here](https://github.com/ikhsanalatsary/multer-sharp/blob/d2671da6c85e4828d82649ccbf223c836a8c308b/test/implementation.test.js#L82)

Reply:
Ah yes! Thank you I should have looked at the test to see all of the possible types. I took a look at the typings for Sharp and Google Cloud Storage and have updated to use them directly instead of needing to define everything on our own. It's possible I may still be missing one or two, I haven't worked much with either of these technologies/libraries.

Reply:
Ok cool